ads/README.md

```diff
@@ -105,9 +105,13 @@ Example usage:
   unlisten();
 ```
 
-##### Initial position
+##### Page layout box
 
-The value `window.context.initialIntersection` contains the initial intersection record at the time the iframe was created.
+The value `window.context.layoutBox` contains the box where the ad is positioned relative to the page.
```

Comment:
Should this be `initialLayoutBox` (both because that is what it is and to align with `window.context.initialIntersection`?